L1Trigger/CSCTriggerPrimitives/src/CSCGEMMatcher.cc

```diff
@@ -38,6 +38,8 @@ unsigned CSCGEMMatcher::calculateGEMCSCBending(const CSCCLCTDigi& clct, const GE
   // difference in 1/8-strip number
   const unsigned diff = std::abs(int(clct.getKeyStrip(8)) - int(cluster.getKeyStrip(8)));
 
+  const bool isME1a(station_ == 1 and clct.getKeyStrip() > CSCConstants::MAX_HALF_STRIP_ME1B);
```

Comment:
```suggestion
```

Reply:
Since this is only used once in this subfunction and duplicates the station 1 requirement, we might as well drop this definition entirely.